spring-cloud-gcp-bigquery/src/main/java/org/springframework/cloud/gcp/bigquery/integration/BigQuerySpringMessageHeaders.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.bigquery.integration;
+
+/**
+ * Spring Integration {@link org.springframework.messaging.Message} headers used with
+ * Spring Cloud GCP BigQuery integration.
+ */
+public final class BigQuerySpringMessageHeaders {
+
+	private static final String PREFIX = "gcp_bigquery_";
```

Comment:
What is wrong to have this as `public` as well?
Imagine I want to have a logic to filter or map all the headers which match a `BigQuerySpringMessageHeaders.PREFIX + '*'` pattern...